be/src/olap/txn_manager.cpp

```diff
@@ -747,8 +747,10 @@ void TxnManager::force_rollback_tablet_related_txns(OlapMeta* meta, TTabletId ta
             }
         }
     }
-    static_cast<void>(
-            RowsetMetaManager::remove_tablet_related_partial_update_info(meta, tablet_id));
+    if (meta != nullptr) {
+        static_cast<void>(
+                RowsetMetaManager::remove_tablet_related_partial_update_info(meta, tablet_id));
+    }
```

Comment:
why cast the result as static void?
If you do not need check the result, I think you should modify the method to return void.